FCMExternalProberTarget/app/src/main/java/com/google/fcmexternalprobertarget/LogReceiver.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.fcmexternalprobertarget;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.widget.TextView;
+
+/**
+ * Receives and handles broadcasts indicating an addition to the in-app logging output
+ */
+public class LogReceiver extends BroadcastReceiver {
+
+    private TextView logTextView;
+
+    /**
+     * Initialize with a given UI element to modify
+     * @param element UI element to be modified upon receiving an appropriate broadcast
+     */
+    public LogReceiver(TextView element) {
+        logTextView = element;
```

Comment:
I would change the name from element to logTextView, and change this line to:
this.logTextView = logTextView;

It's just more idiomatic java style to do it that way.